scripts/wattlab_parity_ofdd_rust_capture.py

```
@@ -0,0 +1,214 @@
+    *,
+    token: str | None,
+    body: dict | None = None,
+) -> tuple[int, dict | list | str]:
+    data = None
+    headers = {"Accept": "application/json"}
+    if token:
+        headers["Authorization"] = f"Bearer {token}"
+    if body is not None:
+        data = json.dumps(body).encode("utf-8")
+        headers["Content-Type"] = "application/json"
+    r = urllib.request.Request(url, data=data, headers=headers, method=method)
```

Ruff (0.16.1)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)